### PR TITLE
Fix multiple re-syncing

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

**Comment by nickmshelley** [comment]
👍  Thanks for fixing my crap! 😄

File: LDSAnnotationsTests/SessionNotificationsTests.swift [hunk 1]
```diff
@@ -30,29 +30,6 @@ class SessionNotificationsTests: XCTestCase {
         XCTAssertEqual(session.status, Session.Status.Unauthenticated)
     }
     
-    func testAuthSuccessNotifications() {
```
Comment (hiltonc) [inline LDSAnnotationsTests/SessionNotificationsTests.swift]: We dropped a test here, would it make sense to replace it, or is it no longer needed?

**Comment by stephanheilner** [comment]
The two that I dropped we to test the status notifications when sign in fails and succeeds. Since we don't update the status for those anymore, the tests wouldn't be helpful. If we add an "authStatus" with observers, then we'll want them again.

**Comment by hiltonc** [comment]
OK, cool. Thanks!

**Pull request merged**